app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidationUserInteraction.java

```diff
@@ -0,0 +1,21 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import java.util.Scanner;
+import java.io.PrintStream;
+
+public class CardValidationUserInteraction {
```

Comment:
For another part of system you will do separate interaction class? Why? interaction is very common thing. And it has very simple logic: read and write only. So it should not be related to the specific business logic now. if you can, you should simplify as much as possible.